### DISCUSSION TITLE
Supabase Bulk Data Upload

### DISCUSSION THREAD

**Discussion by 23000003** [community/community#127950] [Programming Help] [answered]
### Body

Is it possible to upload a bulk data using a CSV/Excel file? 

like example "books data" when a user wants to upload a data, instead of manually inputting every detail, he can just input it all in an csv/excel file and just paste it in the upload functionality.

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by ebndev** [comment, marked as answer]
Hey @23000003, thanks for being a part of the GitHub Community!

When I last used Supabase, you can do this but only on new tables. It might've changed since then though, so check out their documentation: [Import data into Supabase](https://supabase.com/docs/guides/database/import-data) for more details.

If any other folks have done this recently, it would be great to know!

**Reply by 23000003** [reply]
It seems like you can!, i was experimenting and found out that u actually can upload data using csv even tho if its not a new table or the table has data...

**Reply by ebndev** [reply]
That's awesome!
